wdl-ast/examples/explore_requirements.rs

```diff
@@ -0,0 +1,39 @@
+//! Prints `requirements` keys from the first task in a WDL document.
+
+use wdl_ast::AstToken;
+use wdl_ast::Document;
+
+fn main() {
+    let path = std::env::args().nth(1).expect("missing path");
+    let contents = std::fs::read_to_string(path).expect("file to be readable to string");
+
+    let (document, diagnostics) = Document::parse(&contents);
+
+    if !diagnostics.is_empty() {
+        eprintln!(
+            "warning: {n} diagnostics were detected, this may affect results",
+            n = diagnostics.len()
+        )
+    }
+
+    let task = document
+        .ast()
+        // SAFETY: only V1 ASTs are supported for this example at present.
+        .unwrap_v1()
+        .tasks()
+        .next()
+        // SAFETY: we expect the document to contain at least one task for this example
+        // to work correctly.
+        .expect("document to contain at least one task");
+
+    let requirements = task.requirements().next().unwrap_or_else(|| {
```

Comment:
@peterhuene, I did not end up making the changes we discussed for rolling the validation back to the CST here. One of us can do that in a future PR.

Reply:
This is now tracked in #149.